api/rest/index.php

```diff
@@ -70,7 +70,11 @@
 			return $p_response->withStatus( $p_exception->getCode(), $p_exception->getMessage() )->withJson( $t_data );
 		}
 
-		return $p_response->withStatus( HTTP_STATUS_INTERNAL_SERVER_ERROR, $p_exception->getMessage() )->withJson( $t_data );
+		$t_stack_as_string = error_stack_track_as_string( $p_exception );
+		$t_error_to_log =  $p_exception->getMessage() . "\n" . $t_stack_as_string;
+		error_log( $t_error_to_log );
+
+		return $p_response->withStatus( HTTP_STATUS_INTERNAL_SERVER_ERROR )->withJson( $t_data );
```

Comment:
Is it normal to return `->withJson( $t_data )` which still contains the exception's message ?

Reply:
Changed code to include it only if show detailed errors is ON.